browser/ui/views/brave_tab_search_bubble_host.h

```diff
@@ -18,8 +18,11 @@ class BraveTabSearchBubbleHost : public TabSearchBubbleHost {
   void SetBubbleArrow(views::BubbleBorder::Arrow arrow);
 
   // TabSearchBubbleHost:
-  bool ShowTabSearchBubble(bool triggered_by_keyboard_shortcut = false,
-                           int tab_index = -1) override;
+  bool ShowTabSearchBubble(
+      bool triggered_by_keyboard_shortcut = false,
+      int tab_index = -1,
```

Comment:
this should be `SessionID::InvalidValue()`

Reply:
I ended up fixing this in 45c5611ec1ab7487f45eda966205e54855144980.

Reply:
ah, it has another value specifically for tab_search. Great!